noisemaker/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Once deployed, it's a good idea to personalize your site and update these values.